javascript/src/main/components/Courses/CourseTable.js

```diff
@@ -111,6 +111,6 @@ export default ({ courses, admin, deleteCourse }) => {
     }
 
     return (
-        <BootstrapTable keyField='id' data={courses} columns={columns} />
+        <BootstrapTable keyField='id' data={courses} columns={columns} striped={true} />
```

Comment:
`={true}` is redundant in React. If you leave it as `striped`, React will treat this like `striped={true}`.

```suggestion
        <BootstrapTable keyField='id' data={courses} columns={columns} striped />
```